src/usr/local/www/vpn_ipsec_phase1.php

```diff
@@ -750,6 +784,27 @@ function build_eal_list() {
 	    'to allow connections from any IPv6 address.' . '<br/>' . 'Responder Only must be set and ' . 
 	    'Peer IP Address cannot be used for Remote Identifier.'), 'info', false),
 	    '</div>');
+	$group->add(new Form_Input(
+	    'ikeport',
+	    'Remote IKE Port',
+	    'number',
+	    $pconfig['ikeport'],
+	    ['min' => 1, 'max' => 65535]
+	))->setHelp('UDP port for IKE on the remote gateway. Leave empty for default automatic behavior (500/4500).');
+	$group->add(new Form_Input(
+	    'nattport',
+	    'Remote NAT-T Port',
+	    'number',
+	    $pconfig['nattport'],
+	    ['min' => 1, 'max' => 65535]
+	))->setHelp('UDP port for NAT-T on the remote gateway.%1$s%2$s%3$s',
+	    '<div class="infoblock">',
+	    sprint_info_box(gettext('Leave IKE port empty and set NAT-T port if you wish to directly ' . 
```

Comment:
Reword to avoid directly addressing the user ("you") e,.g, "If the IKE port is empty and NAT-T contains a value, the tunnel will use only NAT-T"